src/html_json.js

```diff
@@ -181,12 +157,34 @@ module.exports.main = async (context, action) => {
   const {
     owner, repo, ref, path,
   } = action.request.params;
+
   const { logger } = action;
 
+  const secrets = setdefault(action, 'secrets', {});
+  secrets.REPO_RAW_ROOT = secrets.REPO_RAW_ROOT || 'https://raw.githubusercontent.com/';
+  secrets.HTTP_TIMEOUT = secrets.HTTP_TIMEOUT || 10000;
```

Comment:
Use a default value for `secrets`, not sure, whether there ever is the need for that.